pkg/docker/docker.go

```diff
@@ -665,21 +795,13 @@ func determineCommandBaseDir(opts RunContainerOptions, imageMetadata *docker.Ima
 }
 
 // dumpContainerInfo dumps information about a running container (port/IP/etc).
-func dumpContainerInfo(container *docker.Container, d *stiDocker, image string) {
-	cont, icerr := d.client.InspectContainer(container.ID)
-	liveports := "\n\nPort Bindings:  "
-	if icerr == nil {
-		// Ports is of the following type:  map[docker.Port][]docker.PortBinding
-		for port, bindings := range cont.NetworkSettings.Ports {
-			liveports = liveports + "\n  Container Port:  " + port.Port()
-			liveports = liveports + "\n        Protocol:  " + port.Proto()
-			liveports = liveports + "\n        Public Host / Port Mappings:"
-			for _, binding := range bindings {
-				liveports = liveports + "\n            IP: " + binding.HostIP + " Port: " + binding.HostPort
-			}
-		}
-		liveports = liveports + "\n"
+func dumpContainerInfo(container *dockertypes.ContainerCreateResponse, d *stiDocker, image string) {
+	containerJSON, err := d.kubeDockerClient.InspectContainer(container.ID)
+	if err != nil {
+		return
 	}
+
+	liveports := "\n\nPort Bindings:  \n" + fmt.Sprintf("%+v\n", containerJSON.NetworkSettings.NetworkSettingsBase.Ports)
```

Comment:
ok ... yeah, upon review, I sliced/diced more than needed ...it should be farily close (minus that missing information I mentioned) to what was there before